ui/component/or-asset-viewer/src/index.ts

```diff
@@ -436,11 +440,21 @@ export class OrAssetViewer extends subscribe(manager)(translate(i18next)(LitElem
             }
 
         } else if (panelConfig && panelConfig.type === "chart") {
+
+            if (asset.type === "urn:openremote:asset:group") {
+                return;
+            }
+
```

Comment:
Groups should support chart panels if desired
```suggestion
```

Reply:
Don agrees, fixed.